app/views/providers/limitations/_legacy_proceeding_types.html.erb

```diff
@@ -34,6 +39,27 @@
   <p><strong><%= t('.allowable_work') %></strong><%= application_proceeding_type.substantive_scope_limitation.description %></p>
   </p>
 
+  <p class="govuk-body">
+  <p><%= t('.default_limit_is') %> <strong><%= gds_number_to_currency(@legal_aid_application.default_delegated_functions_cost_limitation,  precision: 0) %></strong></p>
```

Comment:
Does this need to be replicated onto the _multi_proceeding_types.html.erb?
There's a views/providers/limitations/show.html.erb that uses the Setting.allow_multiple_proceedings? and the` if @legal_aid_application.used_delegated_functions?` so maybe there's crossover/duplications there

Reply:
there is a `multi_proceeding_types.html.erb `and a `multi_proceeding_types_with_df.html.erb` this code exists on the `_df `version. I think once we go like with MP we can then remove this `legacy_proceeding_types.html.erb`